rules/windows/image_load/sysmon_in_memory_powershell.yml

```diff
@@ -4,7 +4,7 @@ status: experimental
 description: Detects loading of essential DLL used by PowerShell, but not by the process powershell.exe. Detects meterpreter's "load powershell" extension.
 author: Tom Kern, oscd.community
```

Comment:
Hello @nsaddler! I believe that you can put here your name as well as oscd.community  (:

Reply:
Thank you! Added

Reply:
Thanks!